src/styles/blocks/contact.scss

```diff
@@ -0,0 +1,117 @@
+.contact {
+  @include grid;
+  row-gap: 48px;
+
+  &__title {
+    @include sectionTitle;
+    margin: 0;
+    grid-column: 1/ -1;
+  }
+
+  &__form {
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+
+    grid-column: span 2;
```

Comment:
Will be good to practice how to create separate grid system with separate classes, that will be added to the needed elements in html
But this approach also good, it can just affect on further maintenance